.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,10 @@
+name: "Validate Gradle Wrapper"
```

Comment:
I added a [gradle wrapper validation](https://github.com/gradle/wrapper-validation-action) to ensure that any wrapper jar updates from 3rd parties don't contain malicious code.